umbraco-heartcore/api-documentation/graphql/README.md

```diff
@@ -57,6 +57,39 @@ query {
 }
 ```
 
+## Persisted queries
+
+The GraphQL API supports persisted queries, running a persisted query can be done using the following payload in the Graphql request:
```

Comment:
Suggest we include a brief explanation of what a persisted query actually is, use cases, etc. Could also be worth its own page?

Reply:
@DeadMonkeyX Please have a look at what Jordan is suggesting 😊

Reply:
Hey @jonat123 I've just added a brief explanation of what a persisted query is as Jordan suggested :)

Reply:
@DeadMonkeyX I don't think I see what you added 🤔

Did you maybe forget to push the change to the PR? 😊

Reply:
Hmm strange 🤔  I could have sworn it was pushed correctly. 

I tried again and it seems to be here now :)